Controller/CategoryAdminController.php

```diff
@@ -99,7 +98,7 @@ public function treeAction(Request $request)
         $datagrid = $this->admin->getDatagrid();
 
         if ($this->admin->getPersistentParameter('context')) {
-            $datagrid->setValue('context', ChoiceType::TYPE_EQUAL, $this->admin->getPersistentParameter('context'));
+            $datagrid->setValue('context', null, $this->admin->getPersistentParameter('context'));
```

Comment:
I found:
It is `Sonata\CoreBundle\Form\Type\EqualType` and we should pass there either `EqualType::TYPE_IS_EQUAL` or `EqualType::TYPE_IS_NOT_EQUAL`